cypress/e2e/awx/resources/inventorySource.cy.ts

```diff
@@ -70,7 +77,17 @@ describe('Inventory source page', () => {
     cy.selectDropdownOptionByResourceName('source_control_type', 'Sourced from a Project');
     cy.selectDropdownOptionByResourceName('project', project.name);
     cy.selectDropdownOptionByResourceName('inventory', 'Dockerfile');
-    cy.getBy('.pf-v5-c-input-group > .pf-v5-c-button').click();
+    cy.getBy('[data-cy="execution-environment-select-form-group"]').within(() => {
+      cy.getBy('.pf-v5-c-input-group > .pf-v5-c-button').click();
+    });
+    cy.getBy('[data-ouia-component-type="PF5/ModalContent"]').within(() => {
+      cy.searchAndDisplayResource(executionEnvironmentName);
+      cy.getBy('[data-cy="checkbox-column-cell"] > label').click();
```

Comment:
Our app has a convention in place to use `cy.get`.  I realize this file already has uses of `cy.getBy` though I'm not real sure that is any better.  If we want to be sepecific and get a selector we can use `cy.getBySel` instead.  Otherwise I'd advoated for us to follow existing patterns and just use `cy.get`

Here are the [docs](https://docs.cypress.io/guides/references/best-practices#Real-World-Example)

Reply:
Got it, I was trying to follow precedent in this file but can go through and update all of these.

Reply:
Oh hmmm, looking at the [custom commands](https://github.com/ansible/ansible-ui/blob/d9113fdc0bb0e89f999f141588ece6d85115684f/cypress/COMMANDS.md?plain=1#L5) it seems like we might actually want the "makes sure the element is enabled and visible" features of `cy.getBy` for at least some of the fields in this file? A few fields have `hidden` logic based on the source type and the submit buttons are all disabled until the required data is entered.